code/game/objects/items/implants/implant_loyality.dm

```diff
@@ -20,13 +20,22 @@
 	if(..())
 		if(!target.mind)
 			return TRUE
+
+		var/unbrainwashed = FALSE
+		for(var/datum/objective/brainwashing/B in target.mind.objectives)
```

Comment:
Please isolate these objectives in antag datum. (And if you're doing that make brainwashing machine/abductors use the same one.)